internal/p2p/channel.go

```diff
@@ -61,10 +61,10 @@ func (pe PeerError) Unwrap() error { return pe.Err }
 // Channel is a bidirectional channel to exchange Protobuf messages with peers.
 // Each message is wrapped in an Envelope to specify its sender and receiver.
 type Channel struct {
-	ID    ChannelID
-	In    <-chan Envelope  // inbound messages (peers to reactors)
-	Out   chan<- Envelope  // outbound messages (reactors to peers)
-	Error chan<- PeerError // peer error reporting
+	ID  ChannelID
+	In  <-chan Envelope  // inbound messages (peers to reactors)
+	Out chan<- Envelope  // outbound messages (reactors to peers)
+	err chan<- PeerError // peer error reporting
```

Comment:
Consider maybe `errc` or `errCh` to avert confusion with the common use of `err` for error values (and we seem to use `xCh` as a convention for channels, for better or worse)